test-requirements.txt

```diff
@@ -0,0 +1 @@
+odoo-addon-stock-reserve @ git+https://github.com/OCA/stock-logistics-warehouse.git@refs/pull/1484/head#subdirectory=setup/stock_reserve
```

Comment:
```suggestion
odoo-addon-stock-reserve @ git+https://github.com/OCA/stock-logistics-warehouse.git@refs/pull/1485/head#subdirectory=setup/stock_reserve
```

Link to good PR should work :smiley:

Reply:
Still failling. I tried your suggestion and other change but I can't solve [this](https://github.com/OCA/stock-logistics-warehouse/actions/runs/3271911843/jobs/5382272803#step:3:17).

Thanks

Reply:
Please use suggested syntax. Yours is not correct

Reply:
Thanks for your response but, as I commented before, I tryied your suggestion but [still](https://github.com/OCA/stock-logistics-warehouse/actions/runs/3280454514/jobs/5401247922#step:3:15) failing

I don't find any example. Sorry. Where I can find documentation to read about this?

Reply:
@flachica Not the same error.

Here this is about your module tests. Not not found module.

Reply:
The error about unreleased dependencies is normal. It should be solved before merging this (as the base module is merged).

Reply:
The difference here is you have a runboat running that allows to do functional tests

Reply:
Wow, impressive. I love the functionality. Thanks so much for the help